.env.example

```
@@ -6,3 +6,12 @@ API_PORT=8000
 API_BASE_PATH=/
 # Enable verbose debug logging (true/false)
 DEBUG_MODE=false
+
+# Playwright test configuration
+# Frontend URL (where PHP server is running for tests)
+FRONTEND_URL=http://localhost:3000
+# Path to API repository (defaults to ../LiturgicalCalendarAPI)
+# API_REPO_PATH=/path/to/LiturgicalCalendarAPI
+# Test user credentials (must exist in the API)
+TEST_USERNAME=testuser
+TEST_PASSWORD=testpassword
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TEST_PASSWORD key should go before the TEST_USERNAME key

(UnorderedKey)